data-prepper-plugins/s3-source/src/main/java/org/opensearch/dataprepper/plugins/source/SqsWorker.java

```diff
@@ -168,7 +169,7 @@ private ReceiveMessageRequest createReceiveMessageRequest() {
         return ReceiveMessageRequest.builder()
                 .queueUrl(sqsOptions.getSqsUrl())
                 .maxNumberOfMessages(sqsOptions.getMaximumMessages())
-                .visibilityTimeout((int) sqsOptions.getVisibilityTimeout().getSeconds())
+                .visibilityTimeout(Objects.nonNull(sqsOptions.getVisibilityTimeout()) ? (int) sqsOptions.getVisibilityTimeout().getSeconds() : null)
```

Comment:
Looks good. Can we pass null here?

Reply:
I looked into the code for the `ReceiveMessageRequest` builder and it looks like it should be fine as null. But we still should do some manual testing without the visibility timeout to verify that this uses the queue visibility timeout when null is passed